tests/IntegrationTest.cpp

```diff
@@ -1,247 +0,0 @@
-#include "qx/Random.hpp"
```

Comment:
Shouldn't some of these removed tests be rewritten in cQASM 3, and thereby used to test QX?

Reply:
Right, not all the files and tests have been removed. But `tests` folder has been renamed to `test`.